app/services/ingest_ephemera_csv.rb

```diff
@@ -15,19 +14,16 @@ def initialize(project_id, mdata_file, imgdir, change_set_persister, logger)
 
   def ingest
     mdata_table.collect do |row|
+      change_set = BoxlessEphemeraFolderChangeSet.new(EphemeraFolder.new)
       folder_data = FolderData.new(base_path: imgdir, change_set_persister: change_set_persister, **row.to_h)
       change_set.validate(folder_data.attributes)
       change_set.validate(files: folder_data.files)
-      change_set.validate(append_id: project_id)
-      change_set_persister.save(change_set: change_set)
+      folder_data.member_of_collection_ids.each do |pid|
+        change_set.validate(append_id: pid.id)
+        change_set_persister.save(change_set: change_set)
```

Comment:
I'm not sure I understand what's going on here — can't you do:
```
change_set.validate(member_of_collection_ids: folder_data.member_of_collection_ids)
```
Or would that not save to all the collections correctly?

Reply:
I assume this is adding folders to projects, which is a member relationship and not a collection relationship.

Reply:
Hmm, I think we need to support both — projects are needed for the forms to work, and collections to have multiple collections for DPUL exhibits.

Reply:
How do you suggest I support both?

Reply:
I think the project membership and collection membership need to be handled separately:

For project:
```
change_set.validate(append_id: project_id)
```
For collections:
```
change_set.validate(member_of_collection_ids: folder_data.member_of_collection_ids)
```

Reply:
Various rebases, etc. have confounded the code.  There shouldn't be a project_id parameter, because the projects are included in the spreadsheet/csv file, in conformance with the forms